test/data/manifests/rhel_90-aarch64-ami-boot.json

```diff
@@ -9957,6 +9957,7 @@
         "/etc/pam.d/smartcard-auth": "....L....",
         "/etc/pam.d/system-auth": "....L....",
         "/etc/rhsm/rhsm.conf": "..5....T.",
+        "/etc/ssh/sshd_config": "S.5....T.",
```

Comment:
_Someone_ didn't update the test cases properly :laughing:

Reply:
Totally me. I want a bot for this work. (-:

Reply:
🤖  Tomas at your service 🤣

Reply:
Thanks Tomas! ;) I. All seriousness, this should be done by a bot instead :)

Reply:
I know, I'm joking 😜